src/elm/PageState/Auth/Main.elm

```diff
@@ -511,7 +514,7 @@ viewNavBar model =
                 [ i [ class "icon-plus" ] []
                 , text " New Password"
                 ]
-            , button [ class "newMasterKey btn", onClick OpenNewMasterKeyModal ]
+            , button [ class "newMasterKey btn", disabled (not model.config.masterKeyAllowEdit), onClick OpenNewMasterKeyModal ]
```

Comment:
Alright. Deze kan dan gemerget worden wanneer het mergeconflict is opgelost wat mij betreft.